Assets/MixedRealityToolkit/Inspectors/Profiles/MixedRealityRegisteredServiceProviderProfileInspector.cs

```diff
@@ -153,6 +158,18 @@ private void RenderList(SerializedProperty list)
 
                             serializedObject.ApplyModifiedProperties();
                         }
+
+                        systemType = ((MixedRealityRegisteredServiceProvidersProfile)serializedObject.targetObject).Configurations[i].ComponentType;
+                        if (systemType != null)
+                        {
+                            typeName = systemType.Type?.Name;
+                            providerAttribute = typeName != null ? MixedRealityExtensionServiceAttribute.Find(systemType) : null;
+                        }
+
+                        if (providerAttribute != null && providerAttribute.RequiresProfile && configurationProfile.objectReferenceValue == null)
```

Comment:
Suggestion: move this if check to a dedicated function in the base class inspector?
IsProfileRequired or IsProfileMissing(attribue)?

Reply:
Sounds reasonable. Thanks!

Reply:
done.